docs/includes/google-credentials.md

```diff
@@ -8,7 +8,7 @@
    1. If you don't have an existing project or prefer to have a dedicated one for _fastlane_, [create a new one here](https://console.cloud.google.com/projectcreate/?hl=en) and follow the instructions
 1. Open [Service Accounts on Google Cloud](https://console.cloud.google.com/iam-admin/serviceaccounts?hl=en) and select the project you'd like to use
    1. Click the **CREATE SERVICE ACCOUNT** button at the top of the **Google Cloud Platform Console** page
-   1. Verify that you are on the correct Google Cloud Platform Project by looking for the **Developer Account ID** from earlier within the light gray text in the second input, preceding `.iam.gserviceaccount.com`, or by checking the project name in the navigation bar. If not, open the picker in the top navigation bar, and find the right one.
+   1. Verify that you are on the correct Google Cloud Platform Project by looking for the **Google Cloud Project ID** from earlier within the light gray text in the second input, preceding .iam.gserviceaccount.com, or by checking the project name in the navigation bar. If not, open the picker in the top navigation bar, and find the right one.
```

Comment:
This refers to a previous step on `#L6`, so my guess it both need to be updated at the same time.

Reply:
PS: why remove the code formatting for the namespace?

Reply:
Same question as above re: removal of the code formatting

Reply:
@janbrasna @rogerluan - Apologies for that. Just fixed as per the comments.